README.md

```diff
@@ -218,8 +250,12 @@ Note: if `--backtrace=never` is set, setting `--backtrace-limit` has no effect.
 
 ### "Error: no probe was found."
 
-First, check your data cable:
+First, check your hardware:
 
+- make sure that your development board has an on-board *hardware* debugger.
+If it doesn't, you'll need a separate hardware debugger that works with the JTAG or SWD interface.
+Some boards have no on-board hardware debugger nor expose a JTAG or SWD port and instead only come with *bootloader* firmware that lets you load new program;
```

Comment:
The Raspberry Pi Pico does have an SWD port, but without a debugger plugged into it that's a red herring.

```suggestion
Some boards have a USB micro-B or Type-C connector but only come with *bootloader* firmware that lets you load new program over USB Mass Storage, instead of having a dedicated on-board JTAG or SWD to USB chip;
```